### PR TITLE
Added the cyberSecurity Field and related youtube channel videos.

### LINKED ISSUE #928: Add the cybersecurity field
### Description

I want to add cybersecurity field 
cybersecurity is one of the missing field of study in the table of content of information linksHub is providing 
and also want to add the related content in youtube section .

### Screenshots

Screenshot 2023-06-04 110337

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Owais24code**
Add the cybersecurity field
## Fixes Issue
closes #928 

## Changes proposed

- created a new folder in a database named cyberSecurity

- added files in the cyberSecurity folder 

 1. basic.json
 2. certificate.json
 3. tools.json
 4. platform.json

- created a new file inside the youtube folder

1. named cybersecurity and connected link some related youtube videos of cybersecurity.

- updated the files

1. types/index.ts
2. database/index.ts
3. database/data.ts

 ## Screenshots

### CODE REVIEW

File: database/cyberSecurity/basic.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+[
+    {
+        "name": "THELINUXFOUNDATION",
```
Comment (rupali-codes) [inline database/cyberSecurity/basic.json:3]:
```suggestion
        "name": "The Linux Foundation",
```

File: database/cyberSecurity/basic.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        "description": "THELINUXFOUNDATION  is a website that provides complete knowledge of the linux operating system .",
```
Comment (rupali-codes) [inline database/cyberSecurity/basic.json:4]:
```suggestion
        "description": "The Linux Foundation  is a website that provides complete knowledge of the linux operating system .",
```

File: database/cyberSecurity/basic.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        "url": "https://www.linuxfoundation.org/",
+        "category": "cyberSecurity",
+        "subcategory": "basic"
+    },
+    {
+        "name":"JAVAPOINT-COMPUTERNETWORK",
```
Comment (rupali-codes) [inline database/cyberSecurity/basic.json:10]:
```suggestion
        "name":"Javatpoint Computer Network",
```

File: database/cyberSecurity/basic.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        "description":"JAVAPOINT-COMPUTERNETWORK ,This is a complete Hub for mastering computer network",
```
Comment (rupali-codes) [inline database/cyberSecurity/basic.json:11]:
```suggestion
        "description": "This platform covers all the concepts of computer network from basic-to-advanced level.",
```

File: database/cyberSecurity/basic.json [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        "url":"https://www.javatpoint.com/computer-network-tutorial",
+        "category": "cyberSecurity",
+        "subcategory":"basic"
+    },
+    {
+        "name":"OWASP-TOP-THREAT",
+        "description":"OWASP gives details explaintion of cyber threat and how to rescue from it.",
```
Comment (rupali-codes) [inline database/cyberSecurity/basic.json:18]:
```suggestion
        "description":"Owasp gives a detailed explanation of cyber threats and how to rescue from them.",
```

File: database/cyberSecurity/basic.json [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    },
+    {
+        "name":"JAVAPOINT-COMPUTERNETWORK",
+        "description":"JAVAPOINT-COMPUTERNETWORK ,This is a complete Hub for mastering computer network",
+        "url":"https://www.javatpoint.com/computer-network-tutorial",
+        "category": "cyberSecurity",
+        "subcategory":"basic"
+    },
+    {
+        "name":"OWASP-TOP-THREAT",
```
Comment (rupali-codes) [inline database/cyberSecurity/basic.json:17]:
```suggestion
        "name":"Owasp",
```

File: database/cyberSecurity/basic.json [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+        "description":"OWASP gives details explaintion of cyber threat and how to rescue from it.",
+        "url":"https://owasp.org/Top10/",
+        "category":"cyberSecurity",
+        "subcategory":"basic"
+    },
+    {
+        "name":"Malwarebytes",
```
Comment (rupali-codes) [inline database/cyberSecurity/basic.json:24]:
```suggestion
        "name": "Malware Bytes",
```

File: database/cyberSecurity/basic.json [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+        "description":"Malwarebytes is website that provide information about malware and malicious software.",
```
Comment (rupali-codes) [inline database/cyberSecurity/basic.json:25]:
```suggestion
        "description": "Malware Bytes is website that provides information about malware and malicious software.",
```

File: database/cyberSecurity/basic.json [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+        "url":"https://www.malwarebytes.com/",
+        "category":"cyberSecurity",
+        "subcategory":"basic"
+    },
+    {
+        "name":"CRYPTOGRAPHY-FORTINET",
```
Comment (rupali-codes) [inline database/cyberSecurity/basic.json:31]:
```suggestion
        "name": "Fortinet",
```

Comment (rupali-codes) [inline database/cyberSecurity/basic.json:31]: Please remove this resource as it doesn't provide any specific context

File: database/cyberSecurity/basic.json [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+        "description":"FORTINET website provides information on what is cryptography ? and importance of cryptography in cyber security.",
+        "url":"https://www.fortinet.com/resources/cyberglossary/what-is-cryptography",
+        "category":"cyberSecurity",
+        "subcategory":"basic"
+    },
+    {
+        "name":"Reverse Engineering-TechTarget",
+        "description":"TechTarget gives essential knowledge about Reverse Enginnering and about the stages of reverse engineering",
```
Comment (rupali-codes) [inline database/cyberSecurity/basic.json:39]:
```suggestion
        "description": "TechTarget gives essential knowledge about Reverse Engineering and the stages of reverse engineering",
```

File: database/cyberSecurity/certificate.json [hunk 11]
```diff
@@ -0,0 +1,37 @@
+[
+    {
+        "name":"CompTIA A+",
+        "description":"CompTIA A+ is entry level certificate for the person who is new in cyber security field and it is also most recommend for beginner.",
```
Comment (rupali-codes) [inline database/cyberSecurity/certificate.json:4]:
```suggestion
        "description":"CompTIA A+ is entry level certificate for the person who is new in cyber security field and it is also  recommended for beginner.s",
```

File: database/cyberSecurity/certificate.json [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+        "url":"https://www.comptia.org/certifications/a",
+        "category":"cyberSecurity",
+        "subcategory":"certificate"
+    },
+    {
+        "name":"Cisco CCNA",
+        "description":"Cisco provides a computer network certificate to gain the knowledge of network and tools.",
```
Comment (rupali-codes) [inline database/cyberSecurity/certificate.json:11]:
```suggestion
        "description":"Cisco provides computer network certificates to gain the knowledge of networking and its tools.",
```

File: database/cyberSecurity/certificate.json [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+        "url":"https://www.cisco.com/c/en/us/training-events/training-certifications/certifications/associate/ccna.html",
+        "category":"cyberSecurity",
+        "subcategory":"certificate"
+    },
+    {
+        "name":"Certified Ethical Hacker(CEH)",
+        "description":"Certified Ethical Hacker(CEH)  this certificate is help you gain good hand on skills on Ethical Hacking and knowledge about it .",
```
Comment (rupali-codes) [inline database/cyberSecurity/certificate.json:18]:
```suggestion
        "description":"Certified Ethical Hacker(CEH)  helps you to get good knowledge about Ethical Hacking.",
```

File: database/cyberSecurity/certificate.json [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+        "url":"https://www.eccouncil.org/train-certify/certified-ethical-hacker-ceh/",
+        "category":"cyberSecurity",
+        "subcategory":"certificate"
+    },
+    {
+        "name":"CompTIA Security+",
+        "description":"CompTIA Security+ is mid level certificate for the person who has good basic knowledge in cyber security and it is also most recommend by professional.",
```
Comment (rupali-codes) [inline database/cyberSecurity/certificate.json:25]:
```suggestion
        "description":"CompTIA Security+ is mid level certificate for the person who have basic knowledge of cyber security.",
```

File: database/cyberSecurity/certificate.json [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+        "url":"https://www.comptia.org/certifications/security",
+        "category":"cyberSecurity",
+        "subcategory":"certificate"
+    },
+    {
+        "name":"CISSP",
+        "description":"CISSP is for the person who is interest in cloud security.",
```
Comment (rupali-codes) [inline database/cyberSecurity/certificate.json:32]:
```suggestion
        "description":"This certificate is recommended for the person who is interested in cloud security.",
```

File: database/cyberSecurity/platform.json [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+    {
+        "name":"TryHackMe",
+        "description":"TryHackMe provides a hands on cybersecurity skill experience in real world.",
+        "url":"https://tryhackme.com/",
+        "category":"cyberSecurity",
+        "subcategory":"platform"
+    },
+    {
+        "name":"HackTheBox",
+        "description":"HackTheBox platform provide learning experience with project on real world issue in cyber security and ethical hacking.",
```
Comment (rupali-codes) [inline database/cyberSecurity/platform.json:18]:
```suggestion
        "description":"HackTheBox platform provides learning experience with projects on real world issues in cyber security and ethical hacking.",
```

File: database/cyberSecurity/platform.json [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+        "url":"https://www.hackthebox.com/",
+        "category":"cyberSecurity",
+        "subcategory":"platform"
+    },
+    {
+        "name":"Cybrary",
+        "description":"Cybrary is accessible, affordable training platform provides curated career paths in cybersecurity, threat-informed training, and certification preparation.",
```
Comment (rupali-codes) [inline database/cyberSecurity/platform.json:25]:
```suggestion
        "description":"Cybrary is accessible, affordable training platform that provides curated career paths in cybersecurity, threat-informed training and more",
```

File: database/cyberSecurity/tools.json [hunk 18]
```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "name":"Nmap",
+        "description":"Nmap known as Network Mapper is a free and open source utility for network discovery and security auditing",
```
Comment (rupali-codes) [inline database/cyberSecurity/tools.json:4]:
```suggestion
        "description":"Nmap known as Network Mapper, is a free and open source utility for network discovery and security auditing",
```

File: database/cyberSecurity/tools.json [hunk 19]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        "name":"BurpSuite",
+        "description":"Burp Suite is a tool designed to support and speed up efficiency in penetration testing, CI/CD integration in DevOps with a proper reporting system.",
+        "url":"https://www.softwaretestinghelp.com/burp-suite-tutorial/",
+        "category":"cyberSecurity",
+        "subcategory":"tools"
+    },
+    {
+        "name":"Wireshark",
+        "description":"Wireshark is a network packet analyzer, that capture live packet data from a network interface.",
```
Comment (rupali-codes) [inline database/cyberSecurity/tools.json:18]:
```suggestion
        "description":"Wireshark is a network packet analyzer, that captures live packet data from a network interface.",
```

File: database/cyberSecurity/tools.json [hunk 20]
```diff
@@ -0,0 +1,30 @@
...
+        "url":"https://www.wireshark.org/docs/",
+        "category":"cyberSecurity",
+        "subcategory":"tools"
+    },
+    {
+        "name":"Metasploit",
+        "description":"Metasploit is used as penetration testing tools, is a very powerful all-in-one tool for performing different steps of a penetration test.",
```
Comment (rupali-codes) [inline database/cyberSecurity/tools.json:25]:
```suggestion
        "description":"Metasploit is used as penetration testing tool. Its a very powerful and  all-in-one tool for performing different steps of a penetration test.",
```

File: database/youtube/cybersecurity.json [hunk 21]
```diff
@@ -0,0 +1,26 @@
...
+        "name": "NetworkChuck",
+        "description": "This channel provides free courses on different cybersecurity tools, platform and more.",
+        "url": "https://youtube.com/@NetworkChuck",
+        "category": "youtube",
+        "subcategory": "cybersecurity",
+        "language": "english"
+    },
+    {
+        "name": "Cyberspatial",
+        "description": "This channel shares knowledge about different cybersecurity, network threats and more in cybersecurity realted information.",
```
Comment (rupali-codes) [inline database/youtube/cybersecurity.json:12]:
```suggestion
        "description": "This channel shares knowledge about different cybersecurity, network threats and more in cybersecurity related information.",
```

File: database/youtube/cybersecurity.json [hunk 22]
```diff
@@ -0,0 +1,26 @@
...
+        "url": "https://youtube.com/@Cyberspatial",
+        "category": "youtube",
+        "subcategory": "cybersecurity",
+        "language": "english"  
+    },
+    {
+        "name":"The Cyber Mentor",
+        "description":"This channel provides knowledge about hacking , information security and also had free course related to ethical hacking.",
```
Comment (rupali-codes) [inline database/youtube/cybersecurity.json:20]:
```suggestion
        "description":"This channel provides knowledge about hacking, information security and also have free courses for Ethical Hacking.",
```

File: types/index.ts [hunk 23]
```diff
@@ -36,6 +36,7 @@ export type Category =
   | 'frontend'
   | 'backend'
   | 'ml & ai'
+  | 'cyberSecurity'//owais
```
Comment (rupali-codes) [inline types/index.ts:39]:
```suggestion
  | 'cyberSecurity'
```

**Comment by rupali-codes** [comment]
Also, please format your code and resolve merge conflicts

**Comment by Owais24code** [comment]
@rupali-codes Maam,
I have resolved all the conflicts and made the changes you listed above.
please review them and if they have not been updated please let me know.

File: database/cyberSecurity/basic.json [hunk 24]
```diff
@@ -0,0 +1,44 @@
...
+    {
+        "name":"Javapoint Computer Network",
+        "description":"This platform covers all the concepts of computer network from basic-to-advanced level.",
+        "url":"https://www.javatpoint.com/computer-network-tutorial",
+        "category": "cyberSecurity",
+        "subcategory":"basic"
+    },
+    {
+        "name":"Owasp",
+        "description":"Owasp gives detailed explaintion of cyber threats and how to rescue from them.",
```
Comment (CBID2) [inline database/cyberSecurity/basic.json:18]: [Typos]
```suggestion
        "description":"Owasp gives detailed explanation of cyber threats and how to rescue people from them.",
```

File: database/cyberSecurity/basic.json [hunk 25]
```diff
@@ -0,0 +1,44 @@
...
+    {
+        "name":"CRYPTOGRAPHY-FORTINET",
+        "description":"FORTINET website provides information on what is cryptography ? and importance of cryptography in cyber security.",
+        "url":"https://www.fortinet.com/resources/cyberglossary/what-is-cryptography",
+        "category":"cyberSecurity",
+        "subcategory":"basic"
+    },
+    {
+        "name":"Reverse Engineering-TechTarget",
+        "description": "TechTarget gives essential knowledge about Reverse Engineering and the stages of reverse engineering",
```
Comment (CBID2) [inline database/cyberSecurity/basic.json:39]: [Proofreading]
```suggestion
        "description": "TechTarget gives essential knowledge about Reverse Engineering",
```

File: database/cyberSecurity/certificate.json [hunk 26]
```diff
@@ -0,0 +1,37 @@
+[
+    {
+        "name":"CompTIA A+",
+        "description":"CompTIA A+ is entry level certificate for the person who is new in cyber security field and it is also recommended for beginners.",
```
Comment (CBID2) [inline database/cyberSecurity/certificate.json:4]: [Proofreading]
```suggestion
        "description":"CompTIA A+ provides entry-level certificates for beginners and people who just started their career in Cybersecurity.",
```

File: database/cyberSecurity/certificate.json [hunk 27]
```diff
@@ -0,0 +1,37 @@
...
+        "url":"https://www.comptia.org/certifications/a",
+        "category":"cyberSecurity",
+        "subcategory":"certificate"
+    },
+    {
+        "name":"Cisco CCNA",
+        "description":"Cisco provides computer network certificate to gain the knowledge of networking and its tools.",
```
Comment (CBID2) [inline database/cyberSecurity/certificate.json:11]:
```suggestion
        "description":"Cisco provides computer network certificate to help people gain the knowledge of networking and its tools.",
```

File: database/cyberSecurity/certificate.json [hunk 28]
```diff
@@ -0,0 +1,37 @@
...
+        "url":"https://www.cisco.com/c/en/us/training-events/training-certifications/certifications/associate/ccna.html",
+        "category":"cyberSecurity",
+        "subcategory":"certificate"
+    },
+    {
+        "name":"Certified Ethical Hacker(CEH)",
+        "description":"Certified Ethical Hacker(CEH) helps you to get good knowledge about Ethical Hacking.",
```
Comment (CBID2) [inline database/cyberSecurity/certificate.json:18]: [Question] How does this course teach this topic? Consider using this question to write a more detailed description of this resource.
```suggestion
        "description":"Certified Ethical Hacker(CEH) helps you learn about Ethical Hacking.",
```

File: database/cyberSecurity/certificate.json [hunk 29]
```diff
@@ -0,0 +1,37 @@
...
+        "url":"https://www.eccouncil.org/train-certify/certified-ethical-hacker-ceh/",
+        "category":"cyberSecurity",
+        "subcategory":"certificate"
+    },
+    {
+        "name":"CompTIA Security+",
+        "description":"CompTIA Security+ is mid level certificate for the person who have basic knowledge of cyber security.",
```
Comment (CBID2) [inline database/cyberSecurity/certificate.json:25]:
```suggestion
        "description":"CompTIA Security+ is a mid-level certificate for the person who has basic knowledge about Cybersecurity.",
```

File: database/cyberSecurity/certificate.json [hunk 30]
```diff
@@ -0,0 +1,37 @@
...
+        "url":"https://www.comptia.org/certifications/security",
+        "category":"cyberSecurity",
+        "subcategory":"certificate"
+    },
+    {
+        "name":"CISSP",
+        "description":"This certificate is recommended for the person who is interested in cloud security.",
```
Comment (CBID2) [inline database/cyberSecurity/certificate.json:32]:
```suggestion
        "description":"This certificate is recommended for the person who is interested in learning about Cloud Security.",
```

File: database/cyberSecurity/platform.json [hunk 31]
```diff
@@ -0,0 +1,31 @@
+[
+    {
+        "name":"kali Linux",
+        "description":"Kali Linux is an open-source, Debian-based Linux distribution geared towards various information security tasks, such as Penetration Testing, Security Research, Computer Forensics and Reverse Engineering.",
```
Comment (CBID2) [inline database/cyberSecurity/platform.json:4]:
```suggestion
        "description":"Kali Linux is an open-source, Debian-based Linux distribution product that is geared towards doing various information security tasks, such as Penetration Testing, Security Research, Computer Forensics, and Reverse Engineering.",
```

File: database/cyberSecurity/platform.json [hunk 32]
```diff
@@ -0,0 +1,31 @@
...
+        "url":"https://www.kali.org/",
+        "category":"cyberSecurity",
+        "subcategory":"platform"
+    },
+    {
+        "name":"TryHackMe",
+        "description":"TryHackMe provides a hands on cybersecurity skill experience in real world.",
```
Comment (CBID2) [inline database/cyberSecurity/platform.json:11]:
```suggestion
        "description":"TryHackMe provides people the opportunity to gain real hands-on experience in Cybersecurity.",
```

File: database/cyberSecurity/platform.json [hunk 33]
```diff
@@ -0,0 +1,31 @@
...
+        "url":"https://tryhackme.com/",
+        "category":"cyberSecurity",
+        "subcategory":"platform"
+    },
+    {
+        "name":"HackTheBox",
+        "description":"HackTheBox platform provides learning experience with projects on real world issues in cyber security and ethical hacking.",
```
Comment (CBID2) [inline database/cyberSecurity/platform.json:18]:
```suggestion
        "description":"HackTheBox platform provides users an engaging learning experience with projects that focus on real-world issues in Cybersecurity and Ethical Hacking.",
```

File: database/cyberSecurity/platform.json [hunk 34]
```diff
@@ -0,0 +1,31 @@
...
+        "url":"https://www.hackthebox.com/",
+        "category":"cyberSecurity",
+        "subcategory":"platform"
+    },
+    {
+        "name":"Cybrary",
+        "description":"Cybrary is accessible, affordable training platform that provides curated career paths in cybersecurity, threat-informed training and more.",
```
Comment (CBID2) [inline database/cyberSecurity/platform.json:25]:
```suggestion
        "description":"Cybrary is an accessible, affordable training platform that provides curated career paths in Cybersecurity, threat-informed training, and more.",
```

File: database/cyberSecurity/tools.json [hunk 35]
```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "name":"Nmap",
+        "description":"Nmap known as Network Mapper, is a free and open source utility for network discovery and security auditing.",
```
Comment (CBID2) [inline database/cyberSecurity/tools.json:4]:
```suggestion
        "description":"NMAP, also known as Network Mapper, is a free and open source utility for network discovery and security auditing.",
```

File: database/cyberSecurity/tools.json [hunk 36]
```diff
@@ -0,0 +1,30 @@
...
+        "url":"https://nmap.org/",
+        "category":"cyberSecurity",
+        "subcategory":"tools"
+    },
+    {
+        "name":"BurpSuite",
+        "description":"Burp Suite is a tool designed to support and speed up efficiency in penetration testing, CI/CD integration in DevOps with a proper reporting system.",
```
Comment (CBID2) [inline database/cyberSecurity/tools.json:11]:
```suggestion
        "description":"Burp Suite is a tool designed to support and speed up efficiency in penetration testing and CI/CD integration in DevOps with a proper reporting system.",
```

File: database/cyberSecurity/tools.json [hunk 37]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        "name":"Wireshark",
+        "description":"Wireshark is a network packet analyzer, that captures live packet data from a network interface.",
+        "url":"https://www.wireshark.org/docs/",
+        "category":"cyberSecurity",
+        "subcategory":"tools"
+    },
+    {
+        "name":"Metasploit",
+        "description":"Metasploit is used as penetration testing tools, Its a very powerful and all-in-one tool for performing different steps of a penetration test.",
```
Comment (CBID2) [inline database/cyberSecurity/tools.json:25]:
```suggestion
        "description":"Metasploit is a very powerful all-in-one tool used for performing the different steps of a penetration test.",
```

File: database/youtube/cybersecurity.json [hunk 38]
```diff
@@ -0,0 +1,26 @@
...
+        "name": "NetworkChuck",
+        "description": "This channel provides free courses on different cybersecurity tools, platform and more.",
+        "url": "https://youtube.com/@NetworkChuck",
+        "category": "youtube",
+        "subcategory": "cybersecurity",
+        "language": "english"
+    },
+    {
+        "name": "Cyberspatial",
+        "description": "This channel shares knowledge about different cybersecurity, network threats and more in cybersecurity related information.",
```
Comment (CBID2) [inline database/youtube/cybersecurity.json:12]:
```suggestion
        "description": "This channel shares knowledge about different cybersecurity, network threats, and other cybersecurity-related information.",
```

File: database/youtube/cybersecurity.json [hunk 39]
```diff
@@ -0,0 +1,26 @@
...
+        "url": "https://youtube.com/@Cyberspatial",
+        "category": "youtube",
+        "subcategory": "cybersecurity",
+        "language": "english"  
+    },
+    {
+        "name":"The Cyber Mentor",
+        "description":"This channel provides knowledge about hacking, information security and also have free courses for Ethical Hacking.",
```
Comment (CBID2) [inline database/youtube/cybersecurity.json:20]:
```suggestion
        "description":"This channel provides content about information security and free courses on Ethical Hacking.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Made some proofreading suggestions

**Comment by Owais24code** [comment]
@CBID2 @rupali-codes 
Corrected all proofreading errors and reviewed them.
please do a review.

**Review by rupali-codes — CHANGES_REQUESTED** [review]
Looks perfect though, could you please rename these files then we are ready to go!
* basic.json -> basics.json
* certificate.json -> certification.json
* platform.json -> resources.json _(since we already have resources.json, you may export it as cyberSecurityResources in **index.ts** file)_

Please let us know if you need any kinda help :)

**Comment by Owais24code** [comment]
@rupali-codes  @CBID2 
made changes as you requested please review them and let me know if there are any more changes required to make.
thank you

**Review by rupali-codes — CHANGES_REQUESTED** [review]
can you please remove the checked files, because they are repetitive 
image

**Comment by Owais24code** [comment]
@rupali-codes done! deleted the file that was repetitive.

**Review by CBID2 — CHANGES_REQUESTED** [review]

**Review by CBID2 — DISMISSED** [review]
LGTM

**Review by rupali-codes — COMMENTED** [review]
Please consider removing blank lines :)

File: database/data.ts [hunk 40]
```diff
@@ -60,6 +60,16 @@ export const sidebarData: ISidebar[] = [
       { name: 'deep learning', url: '/deep-learning' },
     ],
   },
+
```
Comment (rupali-codes) [inline database/data.ts:63]:
```suggestion
```

File: database/index.ts [hunk 41]
```diff
@@ -35,6 +35,12 @@ export { default as java} from './languages/java.json'
 export { default as ml } from './ml&ai/ml.json'
 export { default as dataScience } from './ml&ai/datascience.json'
 export { default as deepLearning } from './ml&ai/deeplearning.json'
+// Cyber Security
+export { default as basics} from './cyberSecurity/basics.json'
+export { default as certification} from './cyberSecurity/certification.json'
+export { default as tools} from './cyberSecurity/tools.json'
+export { default as cyberSecurityResources} from './cyberSecurity/resources.json'
+
```
Comment (rupali-codes) [inline database/index.ts:43]:
```suggestion
```

File: types/index.ts
Comment (rupali-codes) [inline types/index.ts:1]: You have incorrectly updated this file, please checkout [this](https://github.com/rupali-codes/LinksHub/blob/main/types/index.ts) carefully.

**Review by rupali-codes — COMMENTED** [review]
Also, the _types_ should be updated as well, based on the previous files we have renamed. 

```sh
subcategory:
basic -> basics
certificate -> certification
platform -> resources
```

let me know if you need any kinda help :)

**Comment by k-deepak04** [comment]
@Owais24code please resolve the conflicts so that we can merge this pr

**Comment by Owais24code** [comment]
@rupali-codes  @k-deepak04 Please review as, I have made all requested changes and resolved all the merge conflicts.

**Comment by Owais24code** [comment]
@k-deepak04 
Screenshot (1)

**Comment by k-deepak04** [comment]
@Owais24code it still has conflicts please resolve those

**Comment by Owais24code** [comment]
> @Owais24code it still has conflicts please resolve those

@k-deepak04 can you please help with this "page is out of dated please start over" issue.

**Comment by Owais24code** [comment]
@rupali-codes Sorry I cannot understand why PR merge has a problem and why vercel showing deployment failed.
can you please help me with this?
Screenshot 2023-07-02 141939

**Comment by k-deepak04** [comment]
> > @Owais24code it still has conflicts please resolve those
> 
> @k-deepak04 can you please help with this "page is out of dated please start over" issue.

@Owais24code I'd be happy to help you with the "page is out of date, please start over" issue. This error typically occurs when the webpage you are trying to access has expired or the information on the page is no longer valid. Here are a few steps you can try to resolve the problem:

Refresh the page: Sometimes, the error message appears due to a temporary issue. Try refreshing the page by pressing the F5 key or clicking the refresh/reload button on your browser.

Clear your browser cache: Cached files can sometimes cause conflicts and display outdated versions of web pages. Clearing your browser's cache can help resolve this issue. The process for clearing the cache varies depending on the browser you're using, but you can usually find the option in the browser's settings or preferences.

Try a different browser: If the issue persists, try accessing the page using a different web browser. This can help determine if the problem is specific to your current browser or if it's a more widespread issue.

Check for updates: Ensure that your browser is up to date. Outdated browser versions can sometimes cause compatibility issues with certain websites. Update your browser to the latest version available.

Contact the website owner: If none of the above steps resolve the issue, it's possible that the website itself is experiencing problems. Reach out to the website owner or administrator and inform them about the error you're encountering. They may be able to provide further assistance or fix the issue on their end.

try this

**Comment by rupali-codes** [comment]
@Owais24code there you go
image

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Review by k-deepak04 — APPROVED** [review]

**Pull request closed without merging**